packages/node/src/workers/cloud-platforms/aws.ts

```diff
@@ -18,7 +18,7 @@ export function spawn(params: WorkerParameters): Promise<WorkerResponse> {
       if (err) {
         reject(err);
       }
-      resolve(data as WorkerResponse);
+      resolve(JSON.parse(JSON.parse(data.Payload as string).body) as WorkerResponse);
```

Comment:
The double parse is pretty weird 🤷🏻‍♂️

Reply:
The problem is that `JSON.parse()` is not deep so
```js
const doublyStringifiedJson = JSON.stringify({name1: JSON.stringify({name2: 'value'})});
console.log(typeof JSON.parse(doublyStringifiedJson).name1);
```
prints `"string"`. Here's a related question:
https://stackoverflow.com/questions/31662849/parse-json-object-that-stringify-twice
We're not stringifying anything explicitly at the functions so I don't see any way around it.